src/main/java/de/unistuttgart/overworldbackend/service/PlayerStatisticService.java

```diff
@@ -61,6 +61,18 @@ public PlayerStatistic getPlayerStatisticFromCourse(final int courseId, final St
             );
     }
 
+    /**
+     * get statistics from a player course
+     *
+     * @param courseId the id of the course
+     * @param userId   the playerId of the player searching for
+     * @return the found playerstatistic
+     * @throws ResponseStatusException (404) when playerstatistic with courseId and userId could not be found
+     */
```

Comment:
```suggestion
    /**
     * Get statistics from a players in a course.
     *
     * @param courseId the id of the course
     * @return the found playerstatistics
     */
```
There is no exception thrown